test/system/src/test/java/io/pravega/test/system/AbstractFailoverTests.java

```diff
@@ -305,6 +305,7 @@ void stopReadersAndWriters(ReaderGroupManager readerGroupManager, String readerG
         testState.writersComplete.get();
         testState.newWritersComplete.get();
         if (testState.getWriteException.get() != null) {
+            log.info("Unable to write events: {}", testState.getWriteException.get());
```

Comment:
Check the documentation:

https://www.slf4j.org/api/org/slf4j/Logger.html#info(java.lang.String,%20java.lang.Throwable)

one of the signatures accepts a Throwable, so it will simply print `{}` in this case, as we can see in the logs.